suite-native/app/src/App.tsx

```diff
@@ -29,6 +33,7 @@ const AppComponent = () => {
     const formattersConfig = useFormattersConfig();
     const isAppReady = useSelector(selectIsAppReady);
     const isConnectInitialized = useSelector(selectIsConnectInitialized);
+    useReportAppInitToAnalytics(APP_STARTED_TIMESTAMP);
```

Comment:
Why are you passing timestamp as param? You can move it to that file.

Reply:
The average app loading time this method measures on my machine is about 500 ms. If I move the const in this specific file it is just about 200 ms (the const is created later). Do you think I should move it anyway?

Reply:
comment added 30f77342c